app/questionnaire/questionnaire_schema.py

```diff
@@ -229,7 +229,7 @@ def _get_answers_by_id(self) -> dict[str, list[ImmutableDict]]:
     def _populate_answer_dependencies(self) -> None:
         for block in self.get_blocks():
             if block["type"] == "CalculatedSummary":
-                self._update_answer_dependencies_for_answers(
+                self._update_answer_dependencies_for_calculated_summary(
```

Comment:
Updated the name as originally I thought it could be used for any answers dependencies, then realised it was a much more specialist situation and at present only viable for calculated_summary. Leaving the name as is could have caused confusion and be reuse by accident

Reply:
Not a blocker. I get your point, but I also find it confusing that it is named after calculated summary but code block doesn't seem to have anything relating to calculated summaries.

Reply:
I did have it inline for a while, maybe the best solution?

Reply:
I think what you have changed is fine for now. We can revisit later if needed.